#try:
 #   raise
  #  print("Como estas")
#except:
 #   print("Hola")
    

# assert expression genera una excepcion del tipo AssertionError
